code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -682,8 +686,9 @@
 	if(prob(5))
 		to_chat(M, "<span class='notice'>[high_message]</span>")
 	M.AdjustParalysis(-4 SECONDS)
-	M.AdjustStunned(-2, FALSE)
-	M.AdjustWeakened(-2, FALSE)
+	M.AdjustStunned(-4 SECONDS)
+	M.AdjustWeakened(-4 SECONDS)
```

Comment:
I changed these while I was in here. missed them from the status effect refactor.